pyproject.toml

```diff
@@ -1,118 +1,41 @@
+
 [project]
-name = "evalml"
-readme = "README.md"
-description = "an AutoML library that builds, optimizes, and evaluates machine learning pipelines using domain-specific objective functions"
-dynamic = ["version"]
-classifiers = [
-    "Development Status :: 4 - Beta",
-    "Intended Audience :: Science/Research",
-    "Intended Audience :: Developers",
-    "Topic :: Software Development",
-    "Topic :: Scientific/Engineering",
-    "Programming Language :: Python",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Operating System :: Microsoft :: Windows",
-    "Operating System :: POSIX",
-    "Operating System :: Unix",
-    "Operating System :: MacOS",
-]
+name = "CheckMate"
+version = "0.1.3"
+description = "CheckMate is an AutoML library which catches and warns of problems with your data and problem setup before modeling."
 authors = [
-    {name="Alteryx, Inc.", email="open_source_support@alteryx.com"}
+    {name = "Nabil Fayak", email = "nabil.fayak@alteryx.com"},
```

Comment:
Do you need this for certain tests to pass right now?

Reply:
The authors section of the pyrpoject.toml? This section isn't needed for tests to pass, was just trying to follow the same format as what I see in the evalml repo, adjusted author when I was using `PDM` to create the pkg, so just entered my details.